internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -224,7 +220,7 @@ func (r *BundleReconciler) ensureUnpackPod(ctx context.Context, bundle *rukpakv1
 			pod.Spec.Containers = make([]corev1.Container, 1)
 		}
 		pod.Spec.Containers[0].Name = bundleUnpackContainerName
-		pod.Spec.Containers[0].Image = bundle.Spec.Image
+		pod.Spec.Containers[0].Image = bundle.Spec.Source.Image.Ref
 		pod.Spec.Containers[0].ImagePullPolicy = corev1.PullAlways
```

Comment:
Unrelated to this PR, but I think we should remove this line so that we inherit the default from the pod spec API, which is:
- If `image` uses a tag and the tag is `:latest`, use `Always`
- Otherwise, use `IfNotPresent`.

Reply:
Yep that's fine with me - made this change in a separate PR: https://github.com/operator-framework/rukpak/pull/165.